test/AWS.Distro.OpenTelemetry.AutoInstrumentation.Tests/AwsMetricAttributesGeneratorTest.cs

```diff
@@ -839,6 +839,36 @@ public void TestSdkClientSpanWithRemoteResourceAttributes()
             { AttributeAWSDynamoTableName, "aws_table_name" },
         };
         this.ValidateRemoteResourceAttributes(attributesCombination, "AWS::DynamoDB::Table", "aws_table_name");
+
+        attributesCombination = new Dictionary<string, object>
```

Comment:
For each test added, please add a case with special chars(^) . Take Python as a reference: https://github.com/aws-observability/aws-otel-python-instrumentation/pull/209/files#diff-38819bf3a39c12774fb5765ee4a38ad2a8dabda81c6c4edf12154f1320f49fa7

Reply:
Added